aim/api/resource.py

```diff
@@ -66,6 +85,8 @@ class BridgeDomain(ResourceBase):
     # Attrbutes completely managed by the DB (eg. timestamps)
     db_attributes = []
 
+    _aci_mo_name = 'fvBD'
```

Comment:
> Honestly speaking we can't paint the whole picture for debugging anyways

Showing the user the whole picture of what *we* control (fvRsCtx in this case) would be possible. But I understand that you want to limit the number of DNs from a single AIM resource to just one. As long as "hiding" the RSs will suffice to reach this goal I'm ok with it. Especially if all the RS DNs are just like fvRsCtx, predictable from the parent.

> Regarding ugliness: in the case of fvBD, fvRsCtx is actually the ugly part.

The ugly part is every ACI specific information IMHO. By confining it to the agent we can eventually enable different backends (or at least that was the claim, at least as long as it doesn't slow down our main goal).

> Exposing the DN of fvRsCtx through AIM will breach the abstraction that AIM is trying to provide, no?

As I said above, I'm OK hiding the RS DNs from Neutron. Even so, I think the answer to this question is no. We break the abstraction if we force the AIM user (Neutron in this case) to understand the ACI specifics, which we are not since Neutron doesn't have to understand DNs (any DN in fact) but just dumbly show them to the human user.

I see the DN as a debug utility for humans/test scripts. Not something that undermines the abstraction. Hypothetically, a backend other than ACI could show something different in the DN field with a similar meaning.